### DISCUSSION TITLE
Modelling the effect of reservoir level rise and fall on landslides

### DISCUSSION THREAD

**Discussion by ljy5590815** [Anura3D/Anura3D_OpenSource#261] [Q&A]
Dear Anura3D devs，
I am a Geological Engineering student and would like to use Anura3D to simulate the effects of reservoir level rise and fall and rainfall on landslides. According to the results made in chapter 13 of the tutorial for rapid water level drop, the change in the diving surface is not normal, is there only one material that can be set up for this type of problem?
Looking forward for someone to answer.
Thanks!

**Comment by Air-shark** [comment]
May I ask if you have made any progress now?I ask want to try it.

**Reply by ljy5590815** [reply]
It can now be run successfully and has made some results, but I don't know if it's right. If you want, we can communicate.

**Reply by Air-shark** [reply]
I'm very glad to communicate with you.I would like to use Anura3D to simulate the effects of reservoir level rise and fall.My email:[402500240051@email.ncu.edu.cn](mailto:402500240051@email.ncu.edu.cn)
